linkerd/app/inbound/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -1,10 +1,10 @@
 #![no_main]
 
 #[cfg(fuzzing)]
-use {libfuzzer_sys::fuzz_target, linkerd_app_inbound::http::fuzz_logic::*};
+use {libfuzzer_sys::fuzz_target, linkerd_app_inbound::http_fuzz};
```

Comment:
the `http` module was public only for this. renamed and re-exported to hide these details